backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -108,17 +108,17 @@ class CHIteratorApi extends IteratorApi with Logging with LogLevelUtil {
       updateNativeMetrics: IMetrics => Unit,
       inputIterators: Seq[Iterator[ColumnarBatch]] = Seq()
   ): Iterator[ColumnarBatch] = {
-    val resIter: GeneralOutIterator = GlutenTimeMetric.millis(pipelineTime) {
-      _ =>
-        val transKernel = new CHNativeExpressionEvaluator()
-        val inBatchIters = new JArrayList[GeneralInIterator](inputIterators.map {
-          iter => new ColumnarNativeIterator(genCloseableColumnBatchIterator(iter).asJava)
-        }.asJava)
-        transKernel.createKernelWithBatchIterator(inputPartition.plan, inBatchIters, false)
-    }
-    TaskContext.get().addTaskCompletionListener[Unit](_ => resIter.close())
+
+    val transKernel = new CHNativeExpressionEvaluator()
```

Comment:
cc @baibaichen, `GlutenTimeMetric.millis` is moved to caller side. Assume it is ok to you.